app.js

```diff
@@ -7,6 +7,13 @@ const govukTemplate = require('hof-govuk-template');
 const hoganExpressStrict = require('hogan-express-strict');
 const expressPartialTemplates = require('express-partial-templates');
 const _ = require('lodash');
+const apiCall = require('./lib/api-call');
```

Comment:
`smartSurveyAPICall` or similar would be a better name, you might have other apis